apidoc/Titanium/Media/AudioPlayer.yml

```diff
@@ -231,51 +264,89 @@ events:
     properties:
       - name: progress
         summary: Current progress, in milliseconds.
+        
+  - name: seek
+    summary: Fired once the [seekToTime](Titanium.Media.AudioPlayer.seek) method completes.
+    properties:
+      - name: finished
+        summary: |        
+            The event for any prior seek request that is still in process will be invoked 
+            immediately with the `finished` parameter set to `false`. 
+            If the new request completes without being interrupted by another seek 
+            request or by any other operation this event will be invoked with 
+            the `finished` parameter set to `true`. 
 properties:
   - name: STATE_BUFFERING
     summary: Audio data is being buffered from the network.
     type: Number
     permission: read-only
+    deprecated:
+        since: "7.2.0"
+        notes: Use <Titanium.Media.AUDIO_STATE_BUFFERING> instead.
 
   - name: STATE_INITIALIZED
     summary: Audio playback is being initialized.
     type: Number
     permission: read-only
+    deprecated:
+        since: "7.2.0"
+        notes: Use <Titanium.Media.AUDIO_STATE_BUFFERING> instead.
```

Comment:
notes  should be changed with relevant state instead of  'Titanium.Media.AUDIO_STATE_BUFFERING'.